conda/api.py

```diff
@@ -49,8 +49,13 @@ def app_get_index(all_version=False):
     By default only the latest version of each app is included in the result,
     unless all_version is set to True.
     """
+    import sys
```

Comment:
According to @ilanschnell most of the functionality in `api` should not be used. The only one that I'm aware of that gets active use is `get_index()` .